### ISSUE TITLE
Tray

### ISSUE THREAD

**Issue by terreng** [terreng/simple-web-server#6]
Windows tray / macOS menu bar icon

**Comment by antnesswcm** [comment]
This is necessary

**Comment by ethanaobrien** [comment]
If I remember correctly, the backend for this issue is finished. I think it just needs to be added as a UI option

**Comment by terreng** [comment]
There's a little bit more to do, and I'd like to have it work on both Windows as well as macOS. I'll try to get around to it soon.

**Comment by terreng** [comment]
@antnesswcm Could you translate this, please?

```
"setting_tray": "Show icon in system tray/menu bar",
"setting_tray_macos": "Show icon in menu bar",
"setting_tray_windows": "Show icon in system tray",
"setting_tray_description": "When enabled, adds a shortcut icon to the system tray or menu bar area whenever Simple Web Server is running.",
```

**Comment by terreng** [comment]
@antnesswcm I just want to make sure you saw this. Thanks!

**Comment by terreng** [comment]
@antnesswcm This was released in version 1.2.6, by the way. Thanks for translating again!
